olive/extra_dependencies.json

```diff
@@ -21,5 +21,8 @@
     ],
     "tf": [
         "tensorflow==1.15.0"
+    ],
```

Comment:
Could you also add references to this extra in this file https://github.com/microsoft/Olive/blob/main/olive/workflows/run/run.py#L59? Sorry, this was a recent change in the repo. Some of the code is still frequently changing so there are new breaking changes.

Reply:
Thanks for your review. I have added references into `run.py`